File: Python_Design-Patterns/Taller_Adapter/Tren.py
from MedioTransporte import *

class Tren(MedioTransporte):

    __costo = 0
    __horario = ''

    def __init__(self):
        print("Creando Tren")

    def getCosto(self):
        return __costo 
    
    def setCosto(self, costo):
        __costo = costo

    def getHorario(self):
        return __horario

    def setHorario(self, horario):
        __horario = horario

File: Java_Design-Patterns/src/talleradapter/Tren.java
package talleradapter;

public class Tren extends MedioTransporte {

    private double costo;
    private String horario;


    public Tren() {
        super();
        System.out.println("Creando Tren");
    }

    @Override
    public double getCosto() {
        return costo;
    }

    @Override
    public void setCosto(double costo) {
        this.costo = costo;
    }

    @Override
    public String getHorario() {
        return horario;
    }

    @Override
    public void setHorario(String horario) {
        this.horario = horario;
    }
}
